src/sarif-converter-21.test.ts

```diff
@@ -87,15 +85,18 @@ describe('SarifConverter21', () => {
         });
 
         it('outputs a sarif log whose run uses the invocationsProvider to populate the invocations property', () => {
-            const stubResults: DecoratedAxeResults = {
+            const stubResults: Axe.AxeResults = {
                 timestamp: stubTimestamp,
-                targetPageUrl: stubTargetPageUrl,
-                targetPageTitle: stubTargetPageTitle,
+                url: stubTargetPageUrl,
                 passes: [],
                 violations: [],
                 inapplicable: [],
                 incomplete: [],
-            };
+                toolOptions: null!,
+                testEngine: null!,
+                testRunner: null!,
+                testEnvironment: null!,
+            } as Axe.AxeResults;
```

Comment:
Do we need to cast `stubResults` as `Axe.AxeResults` if we are already asserting that it is of that type (in `const stubResults: Axe.AxeResults`)?

Reply:
Oops, this was leftover from an earlier attempt at avoiding the `null!`s